extensions/pyRevitTools.extension/pyRevit.tab/Modify.panel/edit2.stack/ReNumber.pushbutton/script.py

```diff
@@ -12,18 +12,22 @@
 output = script.get_output()
 
 
-def toggle_element_selection_handles(target_view, category_name, state=True):
+def toggle_element_selection_handles(target_view, bicat, state=True):
     """Toggle handles for spatial elements"""
-    with revit.Transaction("Toggle {} handles".format(category_name.lower())):
+    with revit.Transaction("Toggle {} handles".format(str(bicat).lower())):
         # if view has template, toggle temp VG overrides
         if state:
             target_view.EnableTemporaryViewPropertiesMode(target_view.Id)
-        rr_cat = revit.query.get_subcategory(category_name, 'Reference')
-        rr_cat.Visible[target_view] = state
-        rr_int = revit.query.get_subcategory(category_name, 'Interior Fill')
-        if not rr_int:
-            rr_int = revit.query.get_subcategory(category_name, 'Interior')
-        rr_int.Visible[target_view] = state
+        try:
+            # subcategories names (e.g. OST_MEPSpaceReferenceVisibility, reduce s)
+            bicat_reference = DB.BuiltInCategory[str(bicat)[:-1] + "ReferenceVisibility"]
```

Comment:
Do not sure if it's correct to take BIC like that, but it works. For emergeny cases I've put it into try/catch. Is there something like `.get(x, None)` for Enums?